lib/models/validationErrors.js

```diff
@@ -0,0 +1,91 @@
+module.exports = {
+    default: {
+        code: 100,
+        error: 'INVALID',
```

Comment:
The `error` and the `code` properties should be related -- they're meant to be string/int versions of the same data, for clients that can't use strings with `switch` statements (e.g. iOS). So each of these should have a unique `error` string that does not conflict with custom errors. Something like `MONOCLE_INCORRECT_TYPE` and `MONOCLE_TOO_SMALL` or whatever else you think may work (perhaps just `M_` prefix?)